base/libgit2/tree.jl

```diff
@@ -1,11 +1,16 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 """
-Traverse the entries in a tree and its subtrees in post or pre order.
+    treewalk(f::Function, tree::GitTree, payload=Any[], post::Bool = false)
+
+Traverse the entries in `tree` and its subtrees in post or pre order.
```

Comment:
What does "post or pre order" mean? Might be good to spell that out in layman's terms.